internal/host/validator.go

```diff
@@ -441,3 +443,37 @@ func (v *validator) printAPIVipConnected(c *validationContext, status validation
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *validator) belongsToMajorityGroup(c *validationContext) validationStatus {
+	if swag.StringValue(c.host.Kind) == models.HostKindAddToExistingClusterHost {
+		return ValidationSuccess
```

Comment:
Maybe adding another status like skipped?

Reply:
@avishayt @ronniel1 what do you think? it's probably not for this PR but i think that it's strange to return successful validation when we didn't validated anything.

Reply:
Currently, in terms of validations, UI shows only failed ones.  Not event pending-for-input.  So, I don't think it is interesting if we are not showing any validations besides the failed ones.

For the state machine, in order to reach state "known",  all validations must be successful.

Reply:
It's fine for now.  We should have another task to add a "not applicable" value.